genon/preprocessor/tests/unit/test_toc_split_unit.py

```
@@ -0,0 +1,547 @@
+    pytest.skip("docling import 불가(환경 의존)", allow_module_level=True)
+
+
+def _make_util(**opts):
+    """do_toc_enrichment=True 로 prompt_manager 가 초기화된 util 생성."""
+    options = DataEnrichmentOptions(do_toc_enrichment=True, **opts)
+    return DocumentEnrichmentUtils(options)
+
+
+def _fake_document(lines):
+    """document.texts 형태의 최소 가짜 문서(페이지 정보 없음)."""
+    return SimpleNamespace(texts=[SimpleNamespace(text=l, prov=[]) for l in lines])
```

Ruff (0.15.17)

[error] Ambiguous variable name: `l`

(E741)